src/documentation/index.md

```diff
@@ -35,7 +35,7 @@ A Beam Runner runs a Beam pipeline on a specific (often distributed) data proces
 * [FlinkRunner]({{ site.baseurl }}/documentation/runners/flink/): Runs on [Apache Flink](http://flink.apache.org).
 * [SparkRunner]({{ site.baseurl }}/documentation/runners/spark/): Runs on [Apache Spark](http://spark.apache.org).
 * [DataflowRunner]({{ site.baseurl }}/documentation/runners/dataflow/): Runs on [Google Cloud Dataflow](https://cloud.google.com/dataflow), a fully managed service within [Google Cloud Platform](https://cloud.google.com/).
-* _[Under Development]_ [ApexRunner]({{ site.baseurl }}/contribute/work-in-progress/#feature-branches): Runs on [Apache Apex](http://apex.apache.org).
+* [ApexRunner]({{ site.baseurl }}/documentation/runners/apex/): Runs on [Apache Apex](http://apex.apache.org).
```

Comment:
Move Apex up, between Direct and Flink? (for fairness, alphabetical sort based on the full name, with the exception of direct runner)